frontend/src/Components/AlgInputs/AlgInputsLoadForecast.js

```diff
@@ -91,19 +93,53 @@ class AlgInputsLoadForecast extends Component {
         this.setState({ open: false});
     };
 
+    update = (field) => {
+        return e => this.setState({ [field]: e.currentTarget.value })
```

Comment:
where is `e` coming from?

Reply:
e is the synthetic event, specifically the on-change event of the current text input field we are updating.

Reply:
Yes, I understand that, my point is, how is that being passed in? It seems like it's implicit, however, the binding of the arrow function looks odd to me and I'm surprised this works.
Here are some relevant resources:
https://reactjs.org/docs/handling-events.html#passing-arguments-to-event-handlers
https://stackoverflow.com/questions/42597602/react-onclick-pass-event-with-parameter

Either way, being explicit is better than implicit in this scenario.

Reply:
ok so something like this, for explicit passing of e: 
`    update = (field, e) => {
        return e => this.setState({ [field]: e.currentTarget.value })
    };
`

Reply:
@jongoncalves and @avpeery I pushed some changes

Reply:
awesome! looks good on my end